### ISSUE TITLE
How can I use a loss as the stopping criteria in Wasserstein GAN?

### ISSUE THREAD

**Issue by RamziFsm** [martinarjovsky/WassersteinGAN#78]
Throughout the training step, the variation of generator loss and critic loss for 1000 epochs as following: 
![a](https://user-images.githubusercontent.com/29406252/94341365-a5682d00-0000-11eb-9b01-9cc2686c6c7d.jpg)
Does this variation look like correct?
How can I use a loss as the stopping criteria in Wasserstein GAN? Can I use Generator loss or critic loss? Can I use the early stopping method?
Thanks
